code/model/SubgoalPolicy.cpp

```diff
@@ -263,13 +263,8 @@ SubgoalPolicy::SubgoalPolicy (void)
 	b_UseComplexNonConnectionFeatures = false;
 }
 
-SubgoalPolicy::~SubgoalPolicy (void)
+void SubgoalPolicy::ResetSentenceConnections ()
```

Comment:
I am not really re-editing that function,
I am just taking out the part which resets the sentence connections so that I can use it in there but also when updating the sentence connections